lib/dotcom_web/components/components.ex

```diff
@@ -10,6 +10,8 @@ defmodule DotcomWeb.Components do
     endpoint: DotcomWeb.Endpoint,
```

Comment:
📓 These were copied over from @thecristen's work. So, these will shake out of this PR.